modules/virt-planning-bare-metal-cluster-for-ocp-virt.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/preparing-to-install-on-bare-metal.adoc
```

Comment:
Also included in the `installing/installing_bare_metal_ipi/ipi-install-prerequisites.adoc` assembly